pandas/tests/io/test_sql.py

```diff
@@ -1820,6 +1813,22 @@ def main(connectable):
         DataFrame({"test_foo_data": [0, 1, 2]}).to_sql("test_foo_data", self.conn)
         main(self.conn)
 
+    def _to_sql_with_negative_npinf(self):
```

Comment:
The function name will need to start with `def test_to_sql...`

And could you write a comment with the github issue number? `# GH ...`